### ISSUE TITLE
error: 'quick_erase' is deprecated

### ISSUE THREAD

**Issue by sjackman** [bcgsc/abyss#173]
```
Making all in ParseAligns
g++ -DHAVE_CONFIG_H -I. -I../../ParseAligns -I..  -I../.. -I../../Common -I/home/sjackman/src/abyss -isystem/Users/sjackman/.homebrew/opt/boost/include -isystem/Users/sjackman/.homebrew/opt/boost/include/include   -isystem/Users/sjackman/.homebrew/opt/google-sparsehash/include  -isystem/home/sjackman/src/abyss/boost_1_56_0 -Wall -Wextra -Werror -g -O2 -MT abyss_fixmate-abyss-fixmate.o -MD -MP -MF .deps/abyss_fixmate-abyss-fixmate.Tpo -c -o abyss_fixmate-abyss-fixmate.o `test -f 'abyss-fixmate.cc' || echo '../../ParseAligns/'`abyss-fixmate.cc
../../ParseAligns/abyss-fixmate.cc:213:7: error: 'quick_erase' is deprecated [-Werror,-Wdeprecated-declarations]
                map.quick_erase(it.first);
                    ^
/Users/sjackman/.homebrew/opt/boost/include/boost/unordered/unordered_map.hpp:723:10: note: 'quick_erase' has been explicitly marked deprecated here
    void quick_erase(const_iterator it) { erase(it); }
         ^
1 error generated.
make[2]: *** [abyss_fixmate-abyss-fixmate.o] Error 1
```

**Comment by sjackman** [comment]
> Boost 1.65.0
Add deprecated attributes to quick_erase and erase_return_void. I really will remove them in a future version this time.

http://www.boost.org/doc/libs/1_65_0/doc/html/unordered/changes.html#unordered.changes.boost_1_65_0

**Comment by sjackman** [comment]
> Boost 1.48.0 - Major update
The container now meets C++11's complexity requirements, but to do so uses a little more memory. This means that quick_erase and erase_return_void are no longer required, they'll be removed in a future version.

http://www.boost.org/doc/libs/1_65_0/doc/html/unordered/changes.html#unordered.changes.boost_1_48_0

**Comment by benvvalk** [comment]
Okay, noted.  I've got a few compiler errors to fix for the 2.0.3 release.  I'll add this one to the list.

**Comment by benvvalk** [comment]
Oops, you've already closed it  :-)
